src/Fields/Relationships/BelongsToMany.php

```diff
@@ -100,11 +100,11 @@ public function onlyCount(): static
         return $this;
     }
 
-    public function inLine(string $separator = '', bool $badge = false, ?Closure $link = null): static
+    public function inLine(string $separator = '', Closure|bool $badge = null, ?Closure $link = null): static
```

Comment:
вы указали что badge может быть или Closure или bool, но присвоили null

```suggestion
    public function inLine(string $separator = '', Closure|bool $badge = false, ?Closure $link = null): static
```